### PR TITLE
[Update] add number of EPS into score mechanism

### PR DESCRIPTION
**Pull request by ckyiricky**
Use deviation between original number of EPS and current number of EPS to calculate 'slice score'
Add slice score to calculate total score

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -196,13 +208,17 @@ func parseResult(file string, outputArray []outputData) (err error) {
 		inZoneTrafficScore := rowData.result.InZoneTraffic * 100
 		// use mean deviation to calcualte deviation score
 		deviationScore := 100.0 - rowData.result.MeanDeviation*100
+		// use number of EndpointSlices deviation to calculate sliceScore
+		numberOfOriginalSlices := math.Ceil(float64(rowData.endpoints) / endpointsPerSlice)
+		sliceScore := 100 - (float64(rowData.endpointSlices)-numberOfOriginalSlices)/numberOfOriginalSlices*100
 		// calculate total score based on two scores above
-		totalScore := 0.6*inZoneTrafficScore + 0.4*deviationScore
+		totalScore := 0.5*inZoneTrafficScore + 0.3*deviationScore + 0.2*sliceScore
```
Comment (robscott) [inline main.go]: at some point we may want to define the weights of the scores as vars at the top of the file so it's easy to tweak them.

**Review by robscott — COMMENTED** [review]
Thanks, this looks great! Only nit is that the commit message should be clearer.

File: main.go (hunk 1 above)
Comment (ckyiricky) [reply main.go]: added const at very beginning.

**Comment by ckyiricky** [comment]
Add number of EndpointSlices into score mechanism:
        Use ratio between original number of EPS and actual number of EPS to calculate the score

**Comment by ckyiricky** [comment]
[Update] Add number of EndpointSlices into score mechanism
Add number of EndpointSlices into score mechanism
        Use ratio between original number of EPS and actual number of EPS to
        calculate the score 
Upload example input and output files

**Review by robscott — APPROVED** [review]
LGTM, thanks!

**Pull request merged**